docker/prod/Jenkinsfile

```diff
@@ -5,6 +5,9 @@ def loggerImage = "artifactory.wikia-inc.com/sus/mediawiki-logger"
 
 def rolloutStatus = 1
 
+// Slack notifications
+env.MW_PUSH_CHANNEL="#mw-prod-deployment"
```

Comment:
I think we need to set `env.MW_SANDBOX_K8S_WEBHOOK` too but this is a secret URL. Those variables should probably be passed from Jenkins.

Reply:
This one is defined in Jenkins (as a "global" env variable there).

Reply:
So how about using this instead?
```
def slackChannel = "#mw-prod-deployment"
```
It's not an environment variable, after all.

Reply:
Good point - e3feb5a